marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -95,13 +97,29 @@ tests = testGroup "Spec.Marconi.ChainIndex.Indexers.Utxo"
           "ToJSON/FromJSON roundtrip for UtxoRow"
           "propJsonRoundtripUtxoRow"
           propJsonRoundtripUtxoRow
+
+    , testGroup "LastSync query"
+        [ testCase
+             "Empty indexer latest sync point is ChainPointAtGenesis"
+             testLastSyncOnFreshIndexer
+
+        , testPropertyNamed
+             "The latest chain point is correct on a indexer that has processed at least a block"
+             "propLastChainPointOnRunningIndexer"
+             propLastChainPointOnRunningIndexer
+
+        , testPropertyNamed
+             "The latest chain point is correct on a indexer that was resumed"
```

Comment:
Maybe a description along the lines of: "When resuming an indexer to a given point, the latest chain point should be that same point"?